x-pack/plugins/observability_solution/observability/public/application/index.tsx

```diff
@@ -86,6 +86,65 @@ export const renderApp = ({
   const CloudProvider = plugins.cloud?.CloudContextProvider ?? React.Fragment;
   const PresentationContextProvider = plugins.presentationUtil?.ContextProvider ?? React.Fragment;
 
+  const clearScreenContext = plugins.observabilityAIAssistant.service.setScreenContext({
+    starterPrompts: [
+      {
+        title: i18n.translate(
+          'xpack.observability.aiAssistant.starterPrompts.doIHaveAlerts.title',
+          {
+            defaultMessage: 'Alerting',
+          }
+        ),
+        prompt: i18n.translate(
+          'xpack.observability.aiAssistant.starterPrompts.doIHaveAlerts.prompt',
+          {
+            defaultMessage: 'Do I have any alerts?',
+          }
+        ),
+        icon: 'bell',
+      },
+      {
+        title: i18n.translate(
+          'xpack.observability.aiAssistant.starterPrompts.howCanICreateANewRule.title',
+          {
+            defaultMessage: 'Rule creation',
+          }
+        ),
+        prompt: i18n.translate(
+          'xpack.observability.aiAssistant.starterPrompts.howCanICreateANewRule.prompt',
+          {
+            defaultMessage: 'How can I create a new rule?',
+          }
+        ),
+        icon: 'createSingleMetricJob',
```

Comment:
Does the Assistant actually know this?

Reply:
What do you mean?

Reply:
How to create a new rule - just curious what the response here is and whether it's helpful.

Reply:
I was about to ask the same. I think these could be great prompt - but we should make sure to train the model so it knows the answer to these either via system prompt or pre-baked knowledge base articles.

Reply:
Yes, they have been reverted to more safe ones provided by TomG.